src/plugins/manualColumnFreeze/manualColumnFreeze.js

```diff
@@ -85,7 +85,10 @@ export class ManualColumnFreeze extends BasePlugin {
   }
 
   /**
-   * Freezes the given column (add it to fixed columns).
+   * Freezes the specified column (i.e. adds it to fixed columns).
```

Comment:
I suppose it's a false-positive error. Maybe there is a way to configure the `eslint-*` package to accept `i.e. .... lower case` syntax 🙈

Reply:
@swistach - should I rephrase it? (we use `i.e.` in other places in docs as well)